aten/src/ATen/BatchedFallback.cpp

```diff
@@ -50,17 +216,21 @@ static bool areAnyArgumentsTensorList(const FunctionSchema& schema) {
 void batchedTensorForLoopFallback(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
   const auto& schema = op.schema();
   const auto num_returns = schema.returns().size();
+
+  if (isInplaceOp(schema)) {
```

Comment:
https://github.com/pytorch/pytorch/issues/45203 may be of interest here

Reply:
Yes, that is definitely useful to avoid recomputation here